GUI/QtQuick/ArmoryServersModel.cpp

```diff
@@ -52,11 +52,14 @@ void ArmoryServersModel::setCurrent (int value)
 void ArmoryServersModel::setData(int curIdx, int connIdx
    , const std::vector<ArmoryServer>& data)
 {
-   QMetaObject::invokeMethod(this, [this, curIdx, connIdx, data] {
+   QMetaObject::invokeMethod(this, [this, &curIdx, connIdx, data] {
       beginResetModel();
       data_ = data;
       endResetModel();
       logger_->debug("[{}] curIdx = {}", __func__, curIdx);
+      if (curIdx == -1) {
+         curIdx = 0;
+      }
```

Comment:
we emit signal currentChanged in void ArmoryServersModel::setCurrent (int value) (below in code) - so it will work. Did you mean this signal?

Reply:
Yes, nice. But this will only happen when you open settings. What about the initial terminal run with clean .ini?

Reply:
In init we will allways show wallet wizard, and I add getArmoryServers to it. But maybe it will be better do in c++ part, check initial launch and call getArmoryServers